rcl/test/rcl/test_timer.cpp

```diff
@@ -141,7 +141,7 @@ TEST_F(TestTimerFixture, test_two_timers) {
     ret = rcl_wait_set_fini(&wait_set);
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
   });
-  ret = rcl_wait(&wait_set, RCL_MS_TO_NS(10));
+  ret = rcl_wait(&wait_set, RCL_MS_TO_NS(100));
```

Comment:
shouldn't this one be less than `time2` period (100ms).

Reply:
Bah, transcoding error.  The timer2 period should be 1000ms.  Will fix.